plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/settings/compiler/JavaCompilerPreferencePresenter.java

```diff
@@ -182,16 +182,6 @@ public void apply(Map<String, String> properties) throws OperationException {
                   createAndAddWidget(option);
                 }
               }
-            })
```

Comment:
Why don't we need this anymore?

Reply:
It annoys when jdt.ls is not ready yet. 
It is always shown at the workspace start.